src/api/routes.ts

```diff
@@ -21,6 +21,27 @@ export const ETHERPAD_ROUTE = `${ITEMS_ROUTE}/etherpad`;
 export const COLLECTIONS_ROUTE = `collections`;
 export const buildAppListRoute = `${APPS_ROUTE}/list`;
 
+export const buildGetOwnItemsRoute = ({
+  page,
+  all,
+  name,
+  limit,
+}: OwnItemsQuery) => {
+  let url = `${GET_OWN_ITEMS_ROUTE}?page=${page}`;
+
+  if (all) {
+    url += `&all=${all}`;
+  }
+  if (name) {
+    url += `&name=${name}`;
+  }
+  if (limit) {
+    url += `&limit=${limit}`;
+  }
```

Comment:
We use a nice utility `qs` that allows you to write:
```js
let url = `${GET_OWN_ITEMS_ROUTE}${qs.stringify({
 page,
 name,
 limit
}, {addPrefix: true})}`; // this might not be the exact name of arg
```
An even better solution would be to use `new URL()` but the link is relative =/